server/routes/employer.routes.js

```diff
@@ -14,18 +14,18 @@ router.post('/',
         .not()
         .isEmpty()
         .withMessage('Please enter your first name')
-        .isAlphanumeric('en-US',{ignore: ' '})
+        .isAlpha('en-US',{ignore: ' '})
         .withMessage('Only characters are allowed for your first and last name')
-        .isLength({ min: 2, max: 30})
+        .isLength({ min: 1, max: 30})
```

Comment:
I updated this to let users use alphabet only, and min length 1 since I googled and found out there are people with 1 character last names